stage1/init/init.go

```diff
@@ -237,60 +237,25 @@ func getArgsEnv(p *stage1commontypes.Pod, flavor string, debug bool, n *networki
 			return nil, nil, fmt.Errorf("flag --private-users cannot be used with an lkvm stage1")
 		}
 
-		// kernel and lkvm are relative path, because init has /var/lib/rkt/..../uuid as its working directory
+		// kernel and hypervisor binaries are relative path, because init has /var/lib/rkt/..../uuid as
```

Comment:
kernel and hypervisor binaries are located relative to the working directory of init (/var/lib/rkt/..../uuid)